tests/test_http_request.py

```diff
@@ -198,10 +197,9 @@ def test_empty_formdata(self):
         r1 = self.request_class("http://www.example.com", formdata={})
         self.assertEqual(r1.body, b'')
 
-    @unittest.skipUnless(six.PY2, "TODO")
     def test_default_encoding(self):
         # using default encoding (utf-8)
-        data = {'one': 'two', 'price': '\xc2\xa3 100'}
+        data = {b'one': b'two', b'price': b'\xc2\xa3 100'}
```

Comment:
No requirement I don't think.
> formdata (dict or iterable of tuples) – is a dictionary (or iterable of (key, value) tuples) containing HTML Form data which will be url-encoded and assigned to the body of the request.

this was to fix the test really: `'\xc2\xa3 100'` needs to be used as bytes input. As unicode, it would return something different from `b'price=%C2%A3+100&one=two'` (as the failed original test shows, correctly I believe)

but worth another test with native strings, I'll add one